Deferred.xcodeproj/project.pbxproj

```diff
@@ -385,6 +391,7 @@
 			baseConfigurationReference = DB32954F1B680744002160CC /* Debug.xcconfig */;
 			buildSettings = {
 				ENABLE_TESTABILITY = YES;
+				ONLY_ACTIVE_ARCH = YES;
```

Comment:
I trust that this is changed from the default for a good reason.  Just calling it out in case it was accidental.